packages/react-components/stable-review/react-components.api.md

```diff
@@ -92,6 +92,8 @@ export interface ChatMessage extends MessageCommon {
     // (undocumented)
     messageType: 'chat';
     // (undocumented)
+    metadata?: Record<string, string>;
```

Comment:
Again, +1 to including this in stable releases already.

(This does mean that our next release will end up being a minor bump)